lib/ansible/modules/windows/win_find.ps1

```diff
@@ -244,13 +244,16 @@ Function Get-FileStat($file) {
     if ($attributes -contains 'ReparsePoint') {
         # TODO: Find a way to differenciate between soft and junction links
         $islnk = $true
-        $isdir = $true
 
-        # Try and get the symlink source, can result in failure if link is broken
-        try {
-            $lnk_source = [Ansible.Command.SymLinkHelper]::GetSymbolicLinkTarget($file)
-            $file_stat.lnk_source = $lnk_source
-        } catch {}
+        if ($attributes -contains 'Directory') {
+            $isdir = $true
+
+            # Try and get the symlink source, can result in failure if link is broken
+            try {
```

Comment:
We should still have this outside of the Directory, you can have a file symbolic link.

Reply:
the original code assumed all links are directories, maybe changing the assumption should come in a diffreant PR?

Reply:
NVM - i see what you mean, ill change it